src/borg/archiver.py

```diff
@@ -446,6 +446,8 @@ def do_create(self, args, repository, manifest=None, key=None):
         matcher = PatternMatcher(fallback=True)
         matcher.add_inclexcl(args.patterns)
 
+        logger.info('Creating archive at "%s"' % args.location.orig)
```

Comment:
would work like this, but maybe better move it to before `if not dry_run:` in line ~520.